packages/hardhat/contracts/ToKanBan.sol

```diff
@@ -40,22 +50,27 @@ contract ToKanBan{
         require(pm == msg.sender,"Only the ");
         _;
     }
+    
 
-    //setting the PM
-    function setPM(address _pm) public {
-        pm = _pm;
-    }
+    // //setting the PM
+    // function setPM(address _pm) public {
+    //     pm = _pm;
+    // }
```

Comment:
You can just remove the code instead of commenting it out, we can always go back into the git history to retrieve older functions if needed.